.github/workflows/e2e.yml

```diff
@@ -0,0 +1,21 @@
+name: e2e
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  test:
+    runs-on: macos-10.15
```

Comment:
So the PR clearly shows that the action works fine, but out of curiosity, what is the reasoning behind running the job on a mac?

Reply:
This machine supports nested virtualization and makes vagrant work via virtualbox.

Reply:
ack, thanks